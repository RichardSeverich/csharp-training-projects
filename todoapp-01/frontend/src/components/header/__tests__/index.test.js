import React from 'react';
import Header from '../.';
import { render } from '@testing-library/react';

describe('components/header', () => {
	it('should be instance of Heading, first child', () => {
		const { container } = render(<Header />);
		expect(container.firstChild).toBeInstanceOf(HTMLHeadingElement);
	});

	it('should be instance of div, second child', () => {
		const { container } = render(<Header />);
		expect(container.querySelector('.header .container')).toBeInstanceOf(HTMLDivElement);
	});
	it('should match the snapshot', () => {
		const { container } = render(<Header />);
		expect(container.firstChild).toMatchSnapshot();
	});
});
