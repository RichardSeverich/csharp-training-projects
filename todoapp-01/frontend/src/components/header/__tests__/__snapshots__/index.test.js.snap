// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/header should match the snapshot 1`] = `
<h1
  class="ui header header-nav"
>
  <div
    class="ui container"
  >
    <a
      href="/"
    >
      ToDo
    </a>
  </div>
</h1>
`;
